source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -71,120 +79,302 @@ msgid "IoT agents: agents for different protocols."
+"indicando qual *tenant* (dentro do token JWT) e qual assunto (``dojot"
+".device-manager.devices``) deseja usar para enviar a mensagem. O "
+"DataBroker chamará o Redis para verificar se este tópico já foi criado e "
+"se o administrador dojot criou um perfil para essa tupla específica "
+"``{tenant, subject}``."
 
 #: ../../source/internal-communication.rst:208
-msgid "The two profile schemes available are shown in :numref:`automatic_scheme` and :numref:`assigned_scheme`."
-msgstr "Os dois esquemas de perfis disponíveis são mostrados na :numref:`automatic_scheme` e na :numref:`assigned_scheme`."
+msgid ""
+"The two profile schemes available are shown in :numref:`automatic_scheme`"
+" and :numref:`assigned_scheme`."
+msgstr ""
+"Os dois esquemas de perfis disponíveis são mostrados na "
+":numref:`automatic_scheme` e na :numref:`assigned_scheme`."
 
 #: ../../source/internal-communication.rst:212
 msgid "Automatic scheme profile"
 msgstr "Perfil de esquema automático"
 
 #: ../../source/internal-communication.rst:221
-msgid "The automatic scheme set the number of Kafka partitions to be used to the topic being created, as well as the replication factor (how many replicas will be there for each topic partition). It's up to Kafka to decide which partition and replica will be assigned to which broker instance. You can check `Kafka partitions and replicas`_ in order to know a bit more about partition and replicas. Of course you can check `Kafka's official documentation`_."
-msgstr "O esquema automático define o número de partições Kafka a serem usadas para o tópico que está sendo criado, bem como o fator de replicação (quantas réplicas estarão disponíveis para cada partição de tópico). Cabe a Kafka decidir qual partição e réplica será atribuída a qual instância do broker. Você pode verificar `Kafka partitions and replicas`_ para conhecer um pouco mais sobre partição e réplicas. Claro que você pode conferir a `Kafka's official documentation`_."
+msgid ""
+"The automatic scheme set the number of Kafka partitions to be used to the"
+" topic being created, as well as the replication factor (how many "
+"replicas will be there for each topic partition). It's up to Kafka to "
+"decide which partition and replica will be assigned to which broker "
+"instance. You can check `Kafka partitions and replicas`_ in order to know"
+" a bit more about partition and replicas. Of course you can check "
+"`Kafka's official documentation`_."
+msgstr ""
+"O esquema automático define o número de partições Kafka a serem usadas "
+"para o tópico que está sendo criado, bem como o fator de replicação "
+"(quantas réplicas estarão disponíveis para cada partição de tópico). Cabe"
+" a Kafka decidir qual partição e réplica será atribuída a qual instância "
+"do broker. Você pode verificar `Kafka partitions and replicas`_ para "
+"conhecer um pouco mais sobre partição e réplicas. Claro que você pode "
+"conferir a `Kafka's official documentation`_."
```

Comment:
Is this really translated when the documentation is generated? I see these links all over the documents but I don't really know if they are translated in the compiled form. Did you check it?